.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "trailingComma": "es5",
+  "singleQuote": false
```

Comment:
Should we just go with the safe-react-apps prettierrc here?
```
{
  "printWidth": 120,
  "trailingComma": "all",
  "singleQuote": true
}
```

Reply:
Especially because the small printWidth on a large screen was always confusing to me so much wasted space.

Reply:
I could get behind two of the three here, I really do not like the appearance of the trailing comma on those react components. But yea, lets give it a try and see how it plays out.

Reply:
Its just SOOO hard to read!

```js
const web3Provider = useMemo(() => new ethers.providers.Web3Provider(new SafeAppProvider(safe, sdk)), [sdk, safe]);
```

Reply:
Just be aware, if we go with that prettier file, that every single project file will now be changing. I'll make a commit to show what its gonna be like.

Reply:
I think its okay. But then again I never saw, that a safe apps project has to follow one specific code-style. At least its not mentioned in the contribution / grant guidelines or am I blind?